### ISSUE TITLE
[BUG]: Devil may cry 3 special edition, ghosting and bloom issue

### ISSUE THREAD

**Issue by LuismaSP89** [PCSX2/pcsx2#5300]
### Describe the Bug

Some ghosting and bloom issues in DMC3 special edition (PAL) it can be seen a lot in the last battle (Mission 20)
The game has also some blue stripes issue, but this can be fixed by setting the half pixel offset to full (Texture) not only vertex. But it doesn't help with the bloom and the gosting.

GS Dumps attached.

Note: Tested with all backends, Vulkan, DX11 & Opengl, all the same.

### Reproduction Steps

Start the last mission of the DMC3, the final battle.

### PCSX2 Revision

1.7.2212

### Operating System

Windows 11

### CPU

i7 4790k

### GPU

GTX 970

### Logs & Dumps

[snaps.zip](https://github.com/PCSX2/pcsx2/files/7835494/snaps.zip)

**Comment by MrCK1** [comment]
It sounds like this only occurs when upscaling, is that correct?

**Comment by LuismaSP89** [comment]
Mmm... seems not, I tested now with ps2 native res and also the same issue.

**Comment by Mrlinkwii** [comment]
is this an issue on latest nightly ?

**Comment by Mrlinkwii** [comment]
can this be rechecked on latest nightly :)

**Comment by LuismaSP89** [comment]
> can this be rechecked on latest nightly :)

This improved a lot on the latest nightly but still present. Also there is a a new issue (tested nightly builds from some months ago and still present), the game has some kind of scanline issue. It looks like and old TV, there are lines on the entire screen.

**Comment by Mrlinkwii** [comment]
@LuismaSP89  can i have a gs dump?

**Comment by LuismaSP89** [comment]
> @LuismaSP89 can i have a gs dump?

Sure. Hold on.

**Comment by LuismaSP89** [comment]
> @LuismaSP89 can i have a gs dump?

Here you go Sir, 3 Gsdump with the scanline problems on it. Let me know if you want any further testing.

[snaps.zip](https://github.com/PCSX2/pcsx2/files/11490548/snaps.zip)

**Comment by LuismaSP89** [comment]
PS: As far as I tested, this is happening also in the other Devil may Cry games, specially on the first one.

**Comment by Mrlinkwii** [comment]
these look fine here 

![image](https://github.com/PCSX2/pcsx2/assets/17685135/acd05b9b-b62b-4c82-a7d6-6307064bdfb8)
![image](https://github.com/PCSX2/pcsx2/assets/17685135/995f664f-1bc3-4b1f-b493-4a9e4b6c0c36)

![image](https://github.com/PCSX2/pcsx2/assets/17685135/68698052-05c0-458d-99a1-7f25a886a533)


did you happen to enable shaders/ change interlacing by accident ?

**Comment by LuismaSP89** [comment]
Yes. That's exactly @Jakey757 said. The artifacts are also present in your screenshots, you can see it doing zoom on the images. I already played with the interlacing options but no one seems perfect for this game, and if it's changed it also break other things.

**Comment by MrCK1** [comment]
The dumps look fine to me with Blend, Bob or None modes. No scanlines whatsoever.

I'm not sure what "blue stripes" are being referred to unless someone can provide an example

**Comment by mirodno** [comment]
The only way to eliminate interleaving artefacts without choosing bob deinterlace is through forcing progressive rendering, which looks crappy on native resolution, but since you're upscaling anyway... 

Insert this into 18C9343F.pnach in "/cheats" folder
```
[Progressive scan (224p)]
author=felixthecat1970
comment=Converts 512x448i into 512x224p
patch=0,EE,201EE060,extended,0000202D
patch=0,EE,201EE264,extended,0000182D
patch=0,EE,00513830,extended,24420004
patch=0,EE,2051368C,extended,AE210048
patch=0,EE,20102C84,extended,3C0700E0
```
Taken from https://github.com/felixthecat1970/gamepatches/blob/main/Playstation_2/Devil%20may%20cry%203%20(Series)/cheats/18C9343F.pnach 
I only copied lines associated with progressive scan, the others are irrelevant in this case (they remove ghosting filter in a few locations)

And bloom issues — it's in-game problem. Special effects seems to be hard-coded to specific locations in 512x448i frames, and even Vertex HPO doesn't fix all of them, like some light sources are still slightly off their intended location (most noticeable on first 2 lamps in the mission 3 bar). It's not taking in count the fact that effects are butchering the further the higher upscale goes. I'm saying all of this because progressive scan makes this problem even more pronounced, so I suggest trying bob-bottom-field instead with high upscale value like 4x. If it's too much for hardware, then progressive scan and 2x. Still looks crappy, but no judder and stuff.

I gave up long time ago for workarounds (all of them either crappy but "kinda working" or working really good until you reach some of specific locations...), and now playin' like back in 00s with native resolution. It's a bad in "calm" scenes with no enemies, but when action hits, it's somehow doable... or even unnoticeable, considering amount of things i need to worry about :^)
Sad, that PAL resolution is broken (or it's a broken deinterlace?), it could make a native res gaming less painful...

Also, it's funny how first DMC got right an opposite of DMC3's problems — special effects are wonderfully upscaled, but the HUD...
